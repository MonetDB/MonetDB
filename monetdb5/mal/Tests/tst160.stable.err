stderr of test 'tst160` in directory 'monetdb5/mal` itself:


# 19:09:43 >  
# 19:09:43 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/log" --set mapi_port=43514 --set sql_port=57655 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst160.mal </dev/null
# 19:09:43 >  


# 19:09:43 >  
# 19:09:43 >  Done.
# 19:09:43 >  

