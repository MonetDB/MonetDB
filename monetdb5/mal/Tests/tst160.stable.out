stdout of test 'tst160` in directory 'monetdb5/mal` itself:


# 19:09:43 >  
# 19:09:43 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/log" --set mapi_port=43514 --set sql_port=57655 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst160.mal </dev/null
# 19:09:43 >  

# Monet Database Server V4.5.0
# Copyright (c) 1993-2004, CWI. All rights reserved.
# Compiled for i686-redhat-linux-gnu/32bit; dynamically linked.
# config:/ufs/mk/monet5/src/mal/Tests/All.conf
# dbfarm:/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_mal
# Visit http://monetdb.cwi.nl/ for further information.
##defining a ptr returning function
#function l():ptr;
#    return l:= 0:ptr;
#end l;
function user.l():ptr;	# 0  (l:ptr)
    return l := 0:ptr;	# 1  (l:ptr)<-(_1:ptr)
end l;	# 2  
#
#z:= l();
#print(z);
#
function user.main():void;	# 0  (main:void)
#defining a ptr returning function 	# 1  (_2:str)
    z := user.l();	# 2  (z:ptr)
    io.print(z);	# 3 IOprint_val (_3:int)<-(z:ptr)
end main;	# 4  
[ 0 ]
#

# 19:09:43 >  
# 19:09:43 >  Done.
# 19:09:43 >  

